chains/evm/contracts/tokenAdminRegistry/TokenPoolFactory/TokenPoolFactory.sol

```diff
@@ -115,8 +115,7 @@ contract TokenPoolFactory is ITypeAndVersion {
     address token = Create2.deploy(0, salt, tokenInitCode);
 
     // Deploy the token pool
-    address pool =
-      _createTokenPool(token, localTokenDecimals, remoteTokenPools, tokenPoolInitCode, salt, PoolType.BURN_MINT);
+    address pool = _createTokenPool(token, localTokenDecimals, remoteTokenPools, tokenPoolInitCode, salt);
```

Comment:
The liquidity flag was removed so now this contract can be simplified. It doesn't need to know pool types as burn mint and lock release have the same constructor args